src/apify/_actor.py

```diff
@@ -231,17 +231,19 @@ async def exit(
         event_listeners_timeout: timedelta | None = EVENT_LISTENERS_TIMEOUT,
         status_message: str | None = None,
         cleanup_timeout: timedelta = timedelta(seconds=30),
+        sys_exit: Callable = sys.exit,
```

Comment:
Can this be passed in when the context manager is used (`async with Actor`)? Otherwise, I'd be more inclined to just not calling `sys.exit` from `Actor.exit`. From my point of view, the proposed solution just lets you do `Actor.exit(sys_exit=something)` instead of `Actor.exit(); something()`.